tasks/EGPSR.tex

```diff
@@ -19,19 +18,12 @@ \subsection*{Focus}
 % Setup
 % %% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \subsection*{Setup}
+Generally assume that the arena is supposed to be in its default state. Therefore, possible problems to solve are:
 \begin{enumerate}
-    \item \textbf{Locations:}
-		\begin{itemize}
-			\item \textbf{Test location:} The task takes place inside the \Arena{}, but some commands may require the robot to go out. The \Arena{} is in its nominal state for this test.
-			\item \textbf{Start location:} The robot starts outside the \Arena{}. When the door opens, the robot moves towards the \textit{Instruction Point}.
-			\item \textbf{Instruction point:} At the beginning of the test, as well as after finishing the first and second command, the robot moves to the \textit{Instruction Point}.
-		\end{itemize}
-	\item \textbf{People:}
+    \item \textbf{Problems:}
 		\begin{itemize}
-			\item \textbf{Operators:} A \emph{Professional Operator} (the referee) commands the robot to execute a task.
-			Optionally, commands can be issued by a \emph{Non-Expert Operator}, i.e. a person from the audience with no robotics background.
-			In this case, the referee gives the goal of the command to the non-expert operator, who will then issue it to the robot in their own words. For example, the generated command might be ``Bring me a coke from the kitchen." then the non-expert operator will be told ``The robot should bring you a coke, which is found in the kitchen.", who then tells the robot ``I want a coke. Go to the kitchen and get me one."
-			If the robot consistently fails to understand the non-expert operator (e.g. after two retries), teams can default to a custom operator.
+			\item \textbf{Objects:} Objects that are not in their default location should be returned there. Objects on the floor are to be thrown in the trash.
+			\item \textbf{Persons:} Any person inside the arena other than the referee and referee assistant have a command for the robot and can be approached.
```

Comment:
What happens when the referee or assistant are approached for a task. Do they say, "no I do not have a task"?

Reply:
People with requests will raise their hands. All other people will say no if approached.